load/type.go

```diff
@@ -89,7 +89,13 @@ func (t *Naked) LoadFields(levels int) error {
 		return nil
 	}
 	for i := 0; i < t.st.NumFields(); i++ {
-		field, err := newField(t, i)
+		m := parseTags(t.st.Tag(i))
```

Comment:
you could move the same logic into `newField` line 60
```
tags := tags.Parse(parent.st.Tag(i))[tagSQLType]
```
And `return nil, nil` if the dash exists, as in line 56.

I think it is nicer to concentrate all the logic of the tags into one place, than injecting it into `newField`.
Sorry for the troubles here, really appreciate the commit :-)

Reply:
No problem, It is just  I have never been in a position to `return nil,nil`

but its okay, will push new commit to fix that.